spec/lib/inclusion_intersection_validator_spec.rb

```diff
@@ -24,7 +24,7 @@ class BadSetupValidator
   class ListValidator
     include ActiveModel::Validations
     attr_accessor :list
-    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], in: VALID_ARRAY, message: "not valid"
+    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], within: VALID_ARRAY, message: "not valid"
   end
 
   subject { List.new }
```

Comment:
I just push a new commit that solve the `subject { ListValidator.new }` and the specs errors... Can you pull the code, and fix the other comments, and then we can merge the PR :)